components/styles.js

```diff
@@ -44,17 +44,16 @@ const styles = StyleSheet.create({
   },
   dayButton: {
     alignItems: 'center',
-    padding: 7,
+    padding: 0,
     width: DEVICE_WIDTH / 7,
+    height: DEVICE_WIDTH / 7,
```

Comment:
Yeah, the design is such that spacing between days are squared, so height should equal width. I'll add height to dayButtonFiller.